plugins/woocommerce-admin/client/products/edit-product-page.tsx

```diff
@@ -2,35 +2,43 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { recordEvent } from '@woocommerce/tracks';
-import { useEffect, useRef } from '@wordpress/element';
-import { useSelect } from '@wordpress/data';
-import { Spinner, FormRef } from '@woocommerce/components';
 import {
+	EXPERIMENTAL_PRODUCT_VARIATIONS_STORE_NAME,
 	PartialProduct,
 	Product,
 	PRODUCTS_STORE_NAME,
 	WCDataSelector,
 } from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+import { useEffect, useRef } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { Spinner, FormRef } from '@woocommerce/components';
 import { useParams } from 'react-router-dom';
 /**
  * Internal dependencies
  */
 import { ProductForm } from './product-form';
 import { ProductFormLayout } from './layout/product-form-layout';
+import { ProductVariationForm } from './product-variation-form';
+import './product-page.scss';
 
 const EditProductPage: React.FC = () => {
```

Comment:
Initially I wanted to create a separate `EditProductVariationPage`.  I'm not 100% sold on this option, but we do need the parent product data anyway so I've handled the variation work alongside the parent product here.

Given this issue is a blocker, I think we merge this if it works well for the time being and we can refactor later if there are more optimal ways to split this code.